src/commands/cp_mv.ts

```diff
@@ -161,7 +162,11 @@ async function getCopyAndMoveOperations(
     }
   } else {
     const fromPath = resolvePath(cwd, fromArgs[0]);
-    const toPath = await lstat(destination).then((p) => p?.isDirectory) ? rustJoin(destination, fromPath) : destination;
```

Comment:
Do we need `rustJoin`? What about line 149?

Reply:
Oh, I think I see what this is for now. As long as we have tests for everything then that's good.

Reply:
What matters most is that it works the same way as sh

Reply:
Its also used in line 87,88

Replacing those with path.join makes the tests fails (but the error message is not informative at all)

I still think that the behavior of rustJoin is more logical

Reply:
Actually replacing rustJoin with path.join(dest,path.basename(src)) seems to keeps the tests working

Reply:
maybe its the same thing, but I'm not confident at all in these path manipulations

Reply:
Actually I'll just remove rustJoin

Reply:
I replaced rustJoin https://github.com/dsherret/dax/pull/87/commits/24e7f34152c8161726aac59aaf8ef815d636837e I think the new function is more intuitive